---
float32[6] sensor_data
int8 err
